File: Python/Medium/طرح عناصر مصفوفتين.py

# Creator: M7ilan
# https://profile.satr.codes/m7ilan/public/overview
# Discord: M7ilan#5185
# https://github.com/M7ilan

def sub_arrays(arr1: list[int], arr2: list[int]) -> list[int]:
    maxLength = 0
    if len(arr1) > len(arr2):
        maxLength = len(arr1)
    else:
        maxLength = len(arr2)
        
    result = []
    for i in range(0, maxLength):
        result.append(arr2[i] - arr1[i])
                      
    return result

File: challenges/JavaScript/Medium/طرح عناصر مصفوفتين.js
/*

Creator: OmarAfet
https://profile.satr.codes/OmarAfet/public/overview
https://github.com/OmarAfet

*/

function sub_arrays(arr1, arr2) {
	let result = [];
	for (let i = 0; i < arr1.length; i++) {
		result.push(arr2[i] - arr1[i]);
	}
	return result;
}
